union_lists.py

```diff
@@ -1,8 +1,8 @@
 def union_lists(list1, list2):
 	list1 = set(list1)
 	list2 = set(list2)
-
+	list3=list((1))
```

Comment:
W191 indentation contains tabs
E225 missing whitespace around operator
F841 local variable 'list3' is assigned to but never used